libs/sdk-ui-kit/src/Icon/icons/InsightIcons/TreeMap.tsx

```diff
@@ -0,0 +1,26 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { IIconProps } from "../../typings";
+
+/**
+ * @internal
+ */
+export const TreeMap: React.FC<IIconProps> = ({ className, width, height, color }) => {
+    const sizes: {
+        width?: number;
+        height?: number;
+    } = {};
+
+    width && (sizes.width = width);
+    height && (sizes.height = height);
+
+    return (
+        <svg {...sizes} className={className} viewBox="0 0 26 26" xmlns="http://www.w3.org/2000/svg">
```

Comment:
isn't this the same as just doing `<svg width={width} height={height} />` as the props are optional?